test/Common/HttpApiHandlerTests.cs

```diff
@@ -919,65 +921,68 @@ public async Task GetEntity_Returns_State_Or_HTTP_404(bool hasKey, bool exists)
-
-            uriBuilder.Path += $"/entities/{entityName}";
+            // Build mock
+            string entityName = Guid.NewGuid().ToString("N");
 
-            var testRequest = new HttpRequestMessage
-            {
-                Method = HttpMethod.Get,
-                RequestUri = uriBuilder.Uri,
-            };
-
-            var list = (IReadOnlyCollection<DurableEntityStatus>)new List<DurableEntityStatus>
+            var mockList = (IReadOnlyCollection<DurableEntityStatus>)new List<DurableEntityStatus>
             {
                 new DurableEntityStatus
                 {
-                    EntityId = new EntityId("DoThis", "one"),
+                    EntityId = new EntityId(entityName, "one"),
                     LastOperationTime = new DateTime(2018, 3, 10, 10, 10, 10, DateTimeKind.Utc),
                     State = null,
                 },
                 new DurableEntityStatus
                 {
-                    EntityId = new EntityId("DoThat", "two"),
+                    EntityId = new EntityId(entityName, "two"),
                     LastOperationTime = new DateTime(2018, 3, 10, 10, 6, 10, DateTimeKind.Utc),
                     State = null,
                 },
             };
 
-            var result = new EntityQueryResult() { Entities = list, ContinuationToken = null };
+            var mockResult = new EntityQueryResult() { Entities = mockList, ContinuationToken = null };
             var clientMock = new Mock<IDurableClient>(MockBehavior.Strict);
 
             clientMock
                     .Setup(x => x.ListEntitiesAsync(It.IsAny<EntityQuery>(), It.IsAny<CancellationToken>()))
-                    .Returns(Task.FromResult(result));
+                    .Returns(Task.FromResult(mockResult));
 
-            var httpApiHandler = new ExtendedHttpApiHandler(clientMock.Object);
+            // Build Uri
+            var uriBuilder = new UriBuilder(TestConstants.NotificationUrl);
+
+            if (hasName)
+            {
+                uriBuilder.Path += $"/entities/{entityName}";
+            }
+            else
+            {
+                uriBuilder.Path += $"/entities/";
+            }
 
             var lastOperationTimeFrom = new DateTime(2018, 3, 10, 10, 1, 0, DateTimeKind.Utc);
             var lastOperationTimeTo = new DateTime(2018, 3, 10, 10, 23, 59, DateTimeKind.Utc);
 
-            var getStatusRequestUriBuilder = new UriBuilder(TestConstants.NotificationUrl);
-            getStatusRequestUriBuilder.Path += $"/Entities/";
-            getStatusRequestUriBuilder.Query = $"lastOperationTimeFrom={WebUtility.UrlEncode(lastOperationTimeFrom.ToString())}&lastOperationTimeTo={WebUtility.UrlEncode(lastOperationTimeTo.ToString())}&fetchInput=false&top=100";
+            uriBuilder.Query = $"lastOperationTimeFrom={WebUtility.UrlEncode(lastOperationTimeFrom.ToString())}&lastOperationTimeTo={WebUtility.UrlEncode(lastOperationTimeTo.ToString())}&fetchInput=false&top=100";

```

Comment:
Per my other feedback, `fetchInput=false` will need to be changed to `fetchState=false` here in the query string.